src/components/containers/visibility.tsx

```diff
@@ -7,7 +7,7 @@ import { eu_domains, uk_domains } from 'common/constants'
 
 type ResponsiveContainerProps = {
     children: ReactElement
-    breakpoint?: number
+    breakpoint?: number | string
```

Comment:
This should be the keys of `size` instead of having it as string which will be too loose.